### DISCUSSION TITLE
Why does tailwind gradient look different from panda css gradients

### DISCUSSION THREAD

**Discussion by satelin2002** [chakra-ui/panda#762] [General]
![image](https://github.com/chakra-ui/panda/assets/54593/a8bfc06b-0e93-4958-b646-427784267f7f)
https://play.tailwindcss.com/jpcOQr83dp 

![image](https://github.com/chakra-ui/panda/assets/54593/af9bb2b2-f1bf-456c-91d3-29a385851418)
<h1
      className={css({
        fontSize: "7xl",
        bgGradient: "to-r",
        gradientFrom: "purple.400",
        gradientTo: "pink.600",
        bgClip: "text",
        fontWeight: "extrabold",
        color: "transparent",
        fontSmoothing: "antialiased",
      })}
    >
      TailwindTailwind
    </h1>

**Comment by anubra266** [comment]
The only difference is that `purple.400` have different values in panda and tailwind's palletes.

https://play.panda-css.com/P6Kh5j_njO
**Tailwind**
![CleanShot 2023-06-20 at 23 09 52@2x](https://github.com/chakra-ui/panda/assets/30869823/a3c5d6ab-601b-4781-ab02-24a6c422d474)
**Panda**
![CleanShot 2023-06-20 at 23 09 44@2x](https://github.com/chakra-ui/panda/assets/30869823/96138093-5360-42b0-af9e-d3cceddd1a76)
